src/Compilers/CSharp/Test/Semantic/Semantics/DynamicTests.cs

```diff
@@ -1275,28 +1401,41 @@ public void TestDynamicArgumentsToCalls()
             string source = @"
 class C
 {
-    public void Goo() {}
-    public void Goo(int x) {}
-    public void Goo(string x) {}
-    public void Goo<T>(int x, int y) where T : class {}
-    public void Goo<T>(string x, string y) where T : class {}
+    public void Goo() { }
+    public void Goo(int x) { }
+    public void Goo(string x) { }
+    public void Goo<T>(int x, int y) where T : class { }
+    public void Goo<T>(string x, string y) where T : class { }
 
     static void M(dynamic d, C c)
     {
         // This could be either of the one-parameter overloads so we allow it.
         c.Goo(d);
 
-        // Doesn't constraints of generic overloads.
-        c.Goo<short>(d, d);
+        /*<bind>*/c.Goo<short>(d, d)/*</bind>*/; // Doesn't constraints of generic overloads.
```

Comment:
What does the comment mean?

Reply:
I am not sure - I have just retained the existing comment. I had to move it outside the `/*<bind>*/` annotation as otherwise the test framework was unable to find the correct syntax node to fetch operation.